am-lib/src/test/java/uk/gov/hmcts/reform/amlib/internal/PermissionsServiceTest.java

```diff
@@ -0,0 +1,282 @@
+package uk.gov.hmcts.reform.amlib.internal;
+
+import com.fasterxml.jackson.core.JsonPointer;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSet;
+import org.junit.jupiter.api.Test;
+import uk.gov.hmcts.reform.amlib.enums.Permission;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.amlib.enums.Permission.CREATE;
+import static uk.gov.hmcts.reform.amlib.enums.Permission.DELETE;
+import static uk.gov.hmcts.reform.amlib.enums.Permission.READ;
+import static uk.gov.hmcts.reform.amlib.enums.Permission.UPDATE;
+
+@SuppressWarnings("PMD")
```

Comment:
1. What is this supressing?
2. Shall we move linelength to class level?

Reply:
Refactored PMD suppressions.

Not sure about line length. On one hand trying to target very small scopes but might not matter in test classes.